core/ast/src/function/async_function.rs

```diff
@@ -259,10 +259,10 @@ impl Spanned for AsyncFunctionExpression {
 impl ToIndentedString for AsyncFunctionExpression {
     fn to_indented_string(&self, interner: &Interner, indentation: usize) -> String {
         let mut buf = "async function".to_owned();
-        if self.has_binding_identifier {
-            if let Some(name) = self.name {
-                let _ = write!(buf, " {}", interner.resolve_expect(name.sym()));
-            }
+        if self.has_binding_identifier
+            && let Some(name) = self.name
```

Comment:
This means the minimum version of Rust we support would be 1.89, but I don't see any changes to rust-version in the crate manifests.